package.json

```diff
@@ -20,9 +20,10 @@
     "start": "web-dev-server --node-resolve --watch --open",
     "test": "npm run lint && npm run test:translations && npm run test:headless && npm run test:axe",
     "test:axe": "web-test-runner --group aXe",
-    "test:headless": "web-test-runner --group default",
-    "test:headless:watch": "web-test-runner --group default --watch",
-    "test:translations": "mfv -e -s en -p ./lang/ -i untranslated"
+    "test:headless": "web-test-runner --group unit",
+    "test:headless:watch": "web-test-runner --group unit --watch",
+    "test:translations": "mfv -e -s en -p ./lang/ -i untranslated",
+    "test:vdiff": "web-test-runner --group vdiff"
```

Comment:
We don't need this line yet though right? Since it won't do anything and isn't called by anything.

Reply:
Yeah, we can do testing with `npx` for now.